src/httpc.c

```diff
@@ -464,6 +464,36 @@ httpc_set_accept_encoding(struct httpc_request *req, const char *encoding)
 #endif
 }
 
+int
+httpc_set_http_version(struct httpc_request *req, const char *version)
+{
+	if (strcmp(version, "1.0") == 0){
```

Comment:
Nit: This code formatting is unusual for the project (`){` should be `) {`). Please, check the code around for examples.

Reply:
Fixed